qdrant-landing/content/documentation/guides/common-errors.md

```diff
@@ -31,3 +31,24 @@ ulimit -n 10000
 ```
 
 Please note, the command should be executed before you run Qdrant server.
+
+## Can't open Collections meta Wal
+
+When starting a Qdrant instance as part of a distributed deployment, you may
+come across an error message similar to this:
+```bash
+Can't open Collections meta Wal: Os { code: 11, kind: WouldBlock, message: "Resource temporarily unavailable" }
+```
+
+It means that Qdrant cannot start because a collection cannot be loaded. Its
+associated [WAL](../../concepts/storage/#versioning) files are currently
+unavailable, likely because the very same files are already being used by
```

Comment:
not sure if we need the very word

Reply:
Used it to emphasize 'same files'. But I removed it now.